collections/_article/godot-openxr-vendors-plugin-400.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Godot OpenXR Vendors Plugin v4"
+excerpt: "What's new in the latest release of the Godot OpenXR Vendors plugin?"
+categories: ["progress-report"]
+author: "David Snopek"
+image: /storage/blog/covers/xxx
+image_caption_description: "Photo by xxx"
+date: 2025-07-23 12:00:00
+---
+
+# Godot OpenXR Vendors Plugin v4
+
+OpenXR support has been built into Godot since the release of Godot 4.0. However, the [OpenXR Vendors plugin](https://github.com/GodotVR/godot_openxr_vendors) (maintained by Godot's XR team) includes extensions to OpenXR created by hardware vendors (e.g. Meta, Pico, HTC, etc.), which we've chosen to keep outside of Godot itself.
+
+We are happy to announce a new major release of the OpenXR Vendors plugin, which includes some exciting new features!
+
+## Version 4.0.0
+
+The 4.0.0 version was actually released right before GodotCon Boston, but with all the excitement around the event (including our [GodotCon presentation](https://godotengine.org/article/godotcon-2025-xr-android-recap/)), we never managed to find a time to properly announce it.
+
+Here's some of the key features in the transition from v3 to v4!
+
+### Switch to Khronos loader
+
+Aside from OpenXR vendor extensions, the plugin has historically also included vendor-specific OpenXR loaders for Android. While OpenXR now has a standard loader for Android (which we call "the Khronos loader"), this wasn't the case when standalone Android headsets first hit the market, and so a number of vendors created their own proprietary loaders.
+
+However, starting with this version 4.0.0, we are only including the Khronos loader, which is supported on all the Android headsets supported by the plugin.
```

Comment:
```suggestion
However, starting with version 4.0.0, we are only including the Khronos loader, which is supported on all the Android headsets supported by the plugin.
```

Reply:
Thanks! I've applied this, and your other suggestions, in my latest push